zipline/assets/asset_writer.py

```diff
@@ -165,6 +165,62 @@ def _generate_output_dataframe(data_subset, defaults):
     return output
 
 
+# Define a version number for the database generated by these writers
+# Increment this version number any time a breaking change is made to the
+# schema and readers of the database
+ASSET_DB_VERSION = 1
```

Comment:
I went back and forth on that - the file is called assets.db, but everything else that references it is fairly uniformly named 'AssetDBBlah' so I stuck with the singular. I could go either way, though.